tools/internal_ci/linux/grpc_binder_transport_apk_build_in_docker.sh

```diff
@@ -18,19 +18,15 @@ set -ex
 echo $ANDROID_HOME
 echo $ANDROID_NDK_HOME
 
-# Build all targets using the strict warning option which leverages the
-# clang compiler to check if sources can pass a set of warning options.
-# CPU are specified because gRPC does not build with 32bit NDK (which has socklen_t
-# defined as int due to an accident).
-# The python option is for disabling python2 enforcement when packing APK
+# Android platforms only works with Bazel >= 7.0
+export OVERRIDE_BAZEL_VERSION=7.0.0
```

Comment:
Is this the right way to force Bazel 7.0?

Reply:
This will break after Bazel 7.0.0 become unsupported, we can simply remove this when we drop support for Bazel < 7.0.0

Reply:
It looks good. Yeah we're going to stop supporting Bazel 6.x ~6 months after Bazel 7 upgrade is done. Then, we can just remove this line.